plover/misc.py

```diff
@@ -55,3 +55,13 @@ def normalize_path(path):
     if path.startswith(ASSET_SCHEME):
```

Comment:
In 1e443041c18c6 commit message the grammar could be improved with "follows": `[...] convert a string as follows`